### PR TITLE
[MISC] prepare release

### PR DESCRIPTION
**Pull request by eseiler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,21 +40,20 @@ The licenses for the applications themselves can be found in the LICENSE files.
...
-Linux, macOS, FreeBSD:
-  * GCC ≥ 5 [limited GCC-4.9 support on Linux]
-  * Clang/LLVM ≥ 3.6 [limited Clang-3.5 support on Linux]
-  * Intel Compiler ≥ 17.0.0 on Linux
-Windows:
-  * Visual C++ ≥ 14.0 / Visual Studio ≥ 2015
-  * Intel Compiler ≥ 17.0.0 / Visual Studio ≥ 2015u2
-  * Clang/C2 ≥ 3.8.0 / Visual Studio ≥ 2015u3 [experimental, requires CMake ≥ 3.6]
+### Linux, macOS, FreeBSD
+  * GCC ≥ 7
```
Comment (smehringer) [inline README.md:44]: I'd still drop a not like
```suggestion
  * GCC ≥ 7 (older versions ≥ 5 might work but are not supported)
```
feel free to shorten.

Also for clang, and intel

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,23 +3,55 @@ SeqAn Changelog
 
 This file summarizes the changes to the SeqAn library and apps.
 
-Release 2.4.1
+Release 2.5.0
 ~~~~~~~~~~~~~
 
+This release aims to add support for interoperability with `SeqAn3 <https://github.com/seqan/seqan3>`_.
```
Comment (smehringer) [inline CHANGELOG.rst:9]:
```suggestion
This release aims to add support for interoperability with `SeqAn3 <https://github.com/seqan/seqan3>`.
```
?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -3,23 +3,55 @@ SeqAn Changelog
...
+Therefore, this release only supports and tests compilers that work with SeqAn3. Release 2.5.0 might work with older
+compilers, but there is no support.
+
+Library Features
+^^^^^^^^^^^^^^^^
+
+- Sequence I/O:
+   - Accepting files that end in `.fas`.
+- Indexing:
+   - Improved search schemes for 3 and 4 errors in bi-FM-Index.
```
Comment (smehringer) [inline CHANGELOG.rst:19]:
```suggestion
   - Improved search schemes for 3 and 4 errors in `seqan::bi_fm_index`.
```

**Review by smehringer — CHANGES_REQUESTED** [review]
:nail_care:

File: CHANGELOG.rst (hunk 2 above)
Comment (eseiler) [reply CHANGELOG.rst:9]: no, that's rst syntax

**Pull request merged**